examples/lighting-app/bouffalolab/README.md

```diff
@@ -0,0 +1,314 @@
+
+## Initial setup
+
+The following steps in this document were validated on Ubuntu 18.04/20.04 and
+Mac OS.
+
+-   Install dependencies as specified in the **connectedhomeip** repository:
+    [Building Matter](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/BUILDING.md).
+
+-   Clone and initialize the **connectedhomeip** repo
+
+    ```
+    git clone https://github.com/project-chip/connectedhomeip.git
+    cd connectedhomeip
+    git submodule update --init --recursive
+    source ./scripts/activate.sh
+    ```
+
+    > After environment setup Bouffalolab flash tool, `bflb-iot-tool`, imports
+    > under this environment. If not, please try `scripts/bootstrap.sh` for
+    > matter environment update.
+
+## Build CHIP Lighting App example
+
+The following steps take examples for BL602 develop board BL602-IoT-Matter-V1
+and BL706 develop board `XT-ZB6-DevKit`.
+
+-   Build lighting app
+
+    ```
+    ./scripts/build/build_examples.py --target bouffalolab-bl602-iot-matter-v1-light build
+    ./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light build
+    ```
+
+-   UART baudrate is 2000000 by default; for baudrate 115200, please build
+    target name with `-115200` appended
+
+    ```shell
+    ./scripts/build/build_examples.py --target bouffalolab-bl602-iot-matter-v1-light-115200 build
+    ./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light-115200 build
+    ```
+
+-   Build target name with `-rpc` appended for rpc enabled as following
+    commands.
+
+    ```shell
+    ./scripts/build/build_examples.py --target bouffalolab-bl602-iot-matter-v1-light-rpc build
+    ./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light-rpc build
+    ```
+
+-   For multiple build options ,such as UART baudrate 115200 + rpc, please try
+
+    ```shell
+    ./scripts/build/build_examples.py --target bouffalolab-bl602-iot-matter-v1-light-rpc-115200 build
+    ./scripts/build/build_examples.py --target bouffalolab-xt-zb6-devkit-light-rpc-115200 build
+    ```
+
+## Download image
+
+-   Using script `*.flash.py`.
+
+    After building gets done, python script
+    `chip-bl602-lighting-example.flash.py` or
+    `chip-bl702-lighting-example.flash.py` will generated under build output
+    folder for BL602 or BL702 building. These python script python uses module
+    `bflb-iot-tool`, make sure terminal is under Matter build environment.
+    Please check `help` option of script for more detail.
+
+    -   Connect the board to your build machine
+
+    -   Put the board to the download mode:
+
+        -   Press and hold the **BOOT** button.
+        -   Click the **RESET** or **EN** button.
+        -   Release the **BOOT** button.
+
+    -   Type following command for image download. Please set serial port
+        accordingly, here we use /dev/ttyACM0 as a serial port example.
+
+        -   `bl602-iot-matter-v1` without additional build options
+
+            ```shel
```

Comment:
should the type of the commands here and below be `shell` instead of `shel` (missing final l) ?

Reply:
Corrected.